### DISCUSSION TITLE
How to add Multiple Homes?

### DISCUSSION THREAD

**Discussion by maverixx** [milo526/homebridge-tuya-web#481] [Q&A] [answered]
Sorry in advance if this is a dumb question. How do I configure TuyaWebPlatform to see devices in _more_ than one home ?
I'm successfully seeing my devices, but only those from one home (e.g. my location in say, Berlin). 

For background info : I have 3 different physical locations successfully operable using Smart Life (e.g. Berlin, London and NYC). These were setup using the "_Home Management/Create home_" option in the Smart Life app.  So, Berlin devices show, but those in London and NYC do not.

It is not clear to me what the next steps are. Do I setup another bridge or child -bridge in Homebridge?  How do I script the config to scan for more than one home?

Thanks in advance, 😀

**Comment by milo526** [comment, marked as answer]
This is a usecase that i have not seen before, and therefor no special support is built-in for this.
I am afraid that the plugin that this API uses does not support this, but you could try to see what the output is from the API as stated on the homepage, with that info I could help you some more.

**Reply by maverixx** [reply]
Thanks for the reply. No problem. Sadly, I kinda expected your answer and it's my own fault for going crazy with home automation 🤪. Thanks also for the kind suggestion to upload data with your follow-on post.  I'm not sufficiently clever to do it, but I appreciate the suggestion. 👍😃

**Comment by milo526** [comment]
Please post the result of [the steps under "How to check whether the api this library uses can control your device".](https://github.com/milo526/homebridge-tuya-web#how-to-check-whether-the-api-this-library-uses-can-control-your-device)

Without this it is hard for me to help you with regards to this specific issue.
